src/templates/templates.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should the `src/templates` directory be called `src/Microsoft.Android.Templates`?